internal/cmd/keys.go

```diff
@@ -77,11 +80,15 @@ $ infra keys add connector
 				return err
 			}
 
-			var userID uid.ID
-			if userName == config.Name { // user is requesting their own stuff
-				userID = config.UserID
-			} else {
-				user, err := getUserByNameOrID(client, userName)
+			userID := config.UserID
+
+			// override the user setting if the user wants to create a connector access key
+			if connector {
+				options.UserName = "connector"
```

Comment:
Right now we use the fixed word `connector` as an argument, so this effectively changes it so that `--connector` replaces it (i.e. `infra keys add --connector` vs `infra keys add connector`).  We don't expose the concept of machine users like we did before (although we may want to revisit that decision at some point), but effectively the connector is just a machine user, so both `--user=connector` and `--connector` will work.